application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/pages/JavaApplicationPage.java

```diff
@@ -263,6 +263,9 @@ protected IXDataRenderer getChartRenderer(IItemCollection itemsInTable, Histogra
 					Messages.JavaApplicationPage_ROW_CPU_USAGE_DESC, allItems, true, JdkQueries.CPU_USAGE_SIMPLE_QUERY,
 					this::isAttributeEnabled, UnitLookup.PERCENT.quantity(0), UnitLookup.PERCENT.quantity(100))
 					.ifPresent(rows::add);
+			DataPageToolkit.buildLinesRow("Threads", "Various thread counts", allItems, true,
```

Comment:
Do you think this might be better left as not filled? At least on Linux, the filled versions aren't transparent so overlaying a couple of these new values quickly covers up whatever is behind it. And with peak and total being the 3rd and 4th options, combining those with either of the first two and it gets hard to read.

example:
With two enabled:
![Screenshot from 2024-11-01 10-33-11](https://github.com/user-attachments/assets/0f63615c-ac5d-4eff-8376-cb3bb8cca0a3)

Peak threads covers up the first two:
![Screenshot from 2024-11-01 10-33-18](https://github.com/user-attachments/assets/f8fb4169-46a9-426b-8774-5e06e4bf171c)

And then total threads covers everything:
![Screenshot from 2024-11-01 10-33-24](https://github.com/user-attachments/assets/4affba34-9b87-4820-b151-d97c132b6def)

Reply:
Good point. Will just use lines.

Reply:
Forgot to localize the strings too.